POS/Operation/03_EditBillList.md

```diff
@@ -3,181 +3,172 @@
 
 # Edit the bill list
 
-The bill list on the right side of the *Cashpoint* tab displays all products added to the list for purchase. Before completing the purchase, you can edit the price and the quantity of products on the list, you can add discounts, remove products from the list, add a sale note or assign the purchase to a new or an existing customer.
+The bill list on the right side of the *Cashpoint* tab displays all offers added to the list for purchase. Before completing the purchase, you can edit the price and the quantity of offers on the list, you can add discounts, remove offers from the list, add a sale note, or assign the purchase to a new or an existing customer.
 
-## Add a discount
 
-You can add a discount to [single products](#add-a-discount-to-a-single-product) or to the [whole purchase](#add-a-discount-to-the-purchase), for instance because of a campaign, defects in the products, for loyal customers or from a certain quantity of products.
+## Add a discount to a single offer
 
-### Add a discount to a single product
+You can add a discount to single offers in your bill list, for instance, because of a campaign, defects in the offers, for loyal customers, or from a certain quantity of offers. 
+The discount is related to the offer entry in the bill list. That means, if the quantity of an offer in the bill list is greater than one, the discount is applied to all offers of this entry. 
```

Comment:
The discount is related to the offer entry in the bill list. That means, if the quantity of an offer in the bill list is greater than one,...
Ich weiß, was du meinst, aber ich finde die Formulierung zu abstrakt bzw. kompliziert.
Wie wäre es: The discount is applied to each offer entry in the bill list. This means that if there is more than one offer in an entry, for example, in case a customer wants to buy (purchase? ;) 3 units of the same item, the discount will be applied to all offers in this entry...? Oder so was.

Reply:
done